contract/models/account_analytic_contract.py

```diff
@@ -60,6 +60,8 @@ class AccountAnalyticContract(models.Model):
         default=lambda s: s._default_journal(),
         domain="[('type', '=', 'sale'),('company_id', '=', company_id)]",
     )
+    company_id = fields.Many2one('res.company', string='Company', required=True,
```

Comment:
Please match the style of the pre-existing fields, with one argument on each line and a trailing comma on the last one.

This style is primarily due to VCS (git) reasons - in order to change or add lines going forward, the diff is not as large & will only touch parts that need to be edited.

Reply:
Thanks lasley. I got it.